locales/en.json

```diff
@@ -1836,5 +1837,187 @@
   },
   "collectionCreatedModal": {
     "title": "Collection Created Successfully"
+  },
+  "errors": {
+    "noPermission": {
+      "name": "No Permission",
+      "description": "You don't have permission to perform this action. Please check your account settings."
+    },
+    "unknownCollection": {
+      "name": "Unknown Collection",
+      "description": "The specified collection doesn't exist. Please verify the collection ID."
+    },
+    "alreadyExists": {
+      "name": "Item Already Exists",
+      "description": "An item with this ID already exists. Please choose a unique ID when minting."
+    },
+    "approvalExpired": {
+      "name": "Approval Expired",
+      "description": "The approval has expired and is no longer valid. Tip: Request a new approval from the item owner."
+    },
+    "wrongOwner": {
+      "name": "Wrong Owner",
+      "description": "The owner information is incorrect. Please double-check the owner details."
+    },
```

Comment:
Instead of mapping all these errors in our projects, is it possible to fetch them from the RPC API call?

Reply:
if there is any better solution, i am all ears

Reply:
You can directly derive it from the error fn

This example is from `useMetaTransaction.ts`

```ts
  const onTxError = async (dispatchError: DispatchError): Promise<void> => {
    const api = await apiInstance.value

    if (dispatchError.isModule) {
      const decoded = api.registry.findMetaError(dispatchError.asModule)
      const { docs, name, section } = decoded
      showNotification(
        `[ERR] ${section}.${name}: ${docs.join(' ')}`,
        notificationTypes.warn,
      )
    } else {
      showNotification(
        `[ERR] ${dispatchError.toString()}`,
        notificationTypes.warn,
      )
    }
  }
  ```

Reply:
that's what I removed to add [this](https://github.com/kodadot/nft-gallery/blob/972b05da4def4ad5feb9e3d96bda3ed5d8db47e1/utils/error.ts#L208)

https://github.com/kodadot/nft-gallery/pull/10096#discussion_r1574832680

## example 

normal error 

![CleanShot 2024-04-23 at 13 56 28@2x](https://github.com/kodadot/nft-gallery/assets/44554284/10b56b6e-6964-47f2-a805-3072ab786b8d)

friendly version 
![CleanShot 2024-04-23 at 13 57 10@2x](https://github.com/kodadot/nft-gallery/assets/44554284/dde426e0-939d-441e-b26a-a3a0e3814dbf)

Reply:
Ok thanks for the info

> Instead of mapping all these errors in our projects

I will try to reuse the errors in EVM version :)